wave-app/src/data.py

```diff
@@ -36,13 +38,15 @@ async def data(q:Q):
     ])
     await q.page.save()
 
-    # Make scatter plot for the first 10000 datapoints.
-    fig = await q.run(make_scatter_plot, q, df[:10000])
+    # Make scatter plot for the 2019-2020 bushfire season.
+    fig = await q.run(show_bush_fires, df)
     # Convert plotly figure to html.
     html = await q.run(to_html, fig)
     # Render figure's html on on the form card.
     q.page['map'] = ui.form_card(box=ui.box('map', order=2), items=[
-        ui.frame(content=html, height='600px')
+        ui.text(BUSHFIRE_INFO),
+        ui.frame(content=html, height='600px'),
+        ui.message_bar(type='info', text=ACKNOWLEDGEMENT),
```

Comment:
just a nitpick -- adding this as a `tooltip` will "beautify" the map card as that text is only visible when you hover over the (i) element